README.md

```diff
@@ -113,25 +105,15 @@ AND  I expect the data to be saved
 Technical Setup
 ===============
 
-You will need to have node and npm installed. This test was verified to work on Node v7.5.0.
-
-1. On the command prompt, run:
-
-    $> cd /path/to/the/test/folder
+You will need to have node and npm installed. This test was verified to work on Node v8.1.4.
 
+On the command prompt, run:
+```
     $> npm install
-    $> npm install -g bower
-    $> bower install
-  
-
-2. Ensure everything was installed successfully
-
-3. Start the environment
-
-    $> npm run-script serve
+    $> npm start
+```
 
-4. Visit the page in browser
+Visit the page in http://localhost:4200
+Profile API can be reached at http://localhost:3000
 
-    http://localhost:3000
 
```

Comment:
Under `Instructions & Deliverables` there's a sentence `You may use any google or other references for angular/bootstrap syntax`. 

I think we should change this to `You may use Google or any other references for angular/bootstrap syntax`